pkg_resources/tests/test_pkg_resources.py

```diff
@@ -236,3 +236,8 @@ def test_version_resolved_from_egg_info(self, env):
         req = pkg_resources.Requirement.parse('foo>=1.9')
         dist = pkg_resources.WorkingSet([env.paths['lib']]).find(req)
         assert dist.version == version
+
+    def test_normalize_path(self, tmpdir):
```

Comment:
In 092ef22, I extracted from your commit just the change for the added test, but as you can see in [this build](https://ci.appveyor.com/project/pypa/setuptools/builds/19789289), even on Windows, that test doesn't capture the failure. I think instead of using `tmpdir` here, you should use a contrived path, something where `pkg_resources.normalize_path()` produces an unexpected result before the patch and an expected result after.